orc8r/cloud/go/obsidian/server/server.go

```diff
@@ -46,27 +45,19 @@ func Start() {
 	e.Use(CollectStats)
 	e.Use(middleware.Recover())
 
-	// With this, we implicitly override the static endpoint serving a static
-	// Swagger spec with a dynamic endpoint which dynamically serves the spec.
-	// Note that this behavior is not documented in the Echo framework.
-	if obsidian.EnableDynamicSwaggerSpecs {
-		yamlCommon, err := swagger.GetCommonSpec()
-		if err != nil {
-			glog.Errorf("Error retrieving Swagger common spec: %+v", err)
-		} else {
-			handler := handlers.GetGenerateCombinedSpecHandler(yamlCommon)
-			e.GET(obsidian.StaticURLPrefix+"/v1/swagger.yml", handler)
-		}
+	err := handlers.RegisterSwaggerHandlers(e)
+	if err != nil {
+		// Swallow RegisterHandlerError because the obsidian service should
+		// continue to run even if Swagger handlers aren't registered.
+		glog.Errorf("Error registering Swagger handlers %+v", err)
 	}
 
-	// Serve static pages for the API docs
-	e.Static(obsidian.StaticURLPrefix, obsidian.StaticFolder+"/apidocs")
-	e.Static(obsidian.StaticURLPrefix+"/swagger-ui/dist", obsidian.StaticFolder+"/swagger-ui/dist")
+	// Serve static assets for the Swagger UI
+	e.Static(obsidian.StaticURLPrefix+"/v1/static/swagger-ui/dist", obsidian.StaticFolder+"/swagger-ui/dist")
```

Comment:
Nit: I think we aligned on having the static assets be *not* scoped by version, so e.g. just `/swagger/static/...`